data/mods/Magiclysm/monsters/holiday_magiclysm.json

```
@@ -0,0 +1,103 @@
+[
+  {
+    "id": "mon_yulecat",
+    "type": "MONSTER",
+    "name": "Yule Cat",
+    "description": " Magical beast from Icelandic folklore, known for their catlike appearance and their hobby of eating Workshop Elves.",
+    "bodytype": "lion",
+    "species": [ "MAGICAL_BEAST" ],
+    "volume": "200000 ml",
+    "weight": "780 kg",
+    "hp": 245,
+    "speed": 167,
+    "material": [ "flesh" ],
+    "symbol": "Y",
+    "color": "white",
+    "aggression": 50,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 7,
+    "melee_dice_sides": 8,
+    "melee_cut": 5,
+    "dodge": 5,
+    "armor_bash": 4,
+    "armor_cut": 1,
+    "armor_bullet": 4,
+    "vision_night": 12,
+    "path_settings": { "max_dist": 21 },
+    "harvest": "mammal_fur",
+    "death_function": [ "NORMAL" ],
+    "reproduction": { "baby_monster": "mon_yulecat_cub", "baby_count": 6, "baby_timer": 234 },
+    "baby_flags": [ "WINTER" ],
+    "special_attacks": [ { "type": "bite", "cooldown": 10 }, [ "GRAB", 8 ], [ "scratch", 6 ] ],
+    "flags": [ "SEES", "HEARS", "SMELLS", "KEENNOSE", "PATH_AVOID_DANGER_1", "WARM", "GRABS" ]
+  },
+  {
+    "id": "mon_yulecat_cub",
+    "type": "MONSTER",
+    "copy-from": "mon_yulecat",
+    "name": { "str": "Yulecat Cub" },
+    "description": "The child of a yulecat, this catlike creature looks as if it poses no threat to you, but looks can be deceiving.",
+    "volume": "120 ml",
+    "weight": "345 g",
+    "hp": 56,
+    "speed": 120,
+    "symbol": "y",
+    "morale": 23,
+    "melee_dice_sides": 6,
+    "melee_cut": 7,
+    "dodge": 6,
+    "harvest": "mammal_fur",
+    "special_attacks": [ [ "EAT_FOOD", 71 ] ],
+    "upgrades": { "age_grow": 234, "into": "mon_yulecat" }
+  },
+  {
+    "id": "mon_workshop_elf",
+    "type": "MONSTER",
+    "name": "Workshop Elf",
```

Needs to manually write down its plural form
```suggestion
    "name": { "str": "Workshop Elf", "str_pl": "Workshop Elves" },
```